homeassistant/components/onkyo/media_player.py

```diff
@@ -197,29 +197,29 @@
 SERVICE_SELECT_HDMI_OUTPUT = "onkyo_select_hdmi_output"
 AUDIO_VIDEO_INFORMATION_UPDATE_INTERVAL = 10
 
-AUDIO_INFORMATION_MAPPING = {
-    "audio_input_port": 0,
-    "input_signal_format": 1,
-    "input_frequency": 2,
-    "input_channels": 3,
-    "listening_mode": 4,
-    "output_channels": 5,
-    "output_frequency": 6,
-    "precision_quartz_lock_system": 7,
-    "auto_phase_control_delay": 8,
-    "auto_phase_control_phase": 9,
-}
+AUDIO_INFORMATION_MAPPING = [
+    "audio_input_port",
+    "input_signal_format",
+    "input_frequency",
+    "input_channels",
+    "listening_mode",
+    "output_channels",
+    "output_frequency",
+    "precision_quartz_lock_system",
+    "auto_phase_control_delay",
+    "auto_phase_control_phase",
+]
 
-VIDEO_INFORMATION_MAPPING = {
-    "video_input_port": 0,
-    "input_resolution": 1,
-    "input_color_schema": 2,
-    "input_color_depth": 3,
-    "output_resolution": 5,
-    "output_color_schema": 6,
-    "output_color_depth": 7,
-    "picture_mode": 8,
-}
+VIDEO_INFORMATION_MAPPING = [
+    "video_input_port",
+    "input_resolution",
+    "input_color_schema",
+    "input_color_depth",
+    "output_resolution",
```

Comment:
You're missing `video_output_port` between `input_color_depth` and `output_resolution`

Reply:
Thanks a lot for reviewing @OnFreund :-)
A bit stupid of me to miss this, but I added it now.

Reply:
Completely understandable - the value wasn't there originally so it was easy to miss.